R/nitrogen_efficiency.R

```diff
@@ -54,6 +54,7 @@ calc_n_efficiency <- function(B_LU_BRP, B_SOILTYPE_AGR, B_GWL_CLASS, B_AER_CBS,
                                                   'Centraal Veehouderijgebied','Oostelijk Veehouderijgebied','Noordelijk Weidegebied',
                                                   'Veenkoloni\u00EBn en Oldambt','Veenkolonien en Oldambt','Bouwhoek en Hogeland'), empty.ok = FALSE)
   checkmate::assert_numeric(B_FERT_NORM_FR, lower = 0, upper = 1, any.missing = FALSE, len = arg.length)
+  checkmate::assert_subset(leaching_to, choices = c('gw', 'ow'))
```

Comment:
If I_H_NOW is renamed to I_H_NSW, why is then 'ow' introduced here, instead of 'sw'?